modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/ParagraphsTestBase.php

```diff
@@ -88,7 +88,7 @@ protected function setUp(): void {
    *
    * @throws \Exception
    */
-  protected function renderParagraph(ParagraphInterface $paragraph, string $langcode = NULL): string {
+  protected function renderParagraph(ParagraphInterface $paragraph, ?string $langcode = NULL): string {
```

Comment:
Doc for this function doesn't have the mixed type parameter, lets keep consistency and add it.
Param description doesn't seem to match: Rendering language code, defaults to 'en'.

Reply:
Union types, not mixed.